spec/API_specification/array_api/fourier_transform_functions.py

```diff
@@ -0,0 +1,468 @@
+        normalization mode. Should be one of the following modes:
+
+        - ``'backward'``: no normalization.
+        - ``'ortho'``: normalize by ``1/sqrt(n)`` (i.e., make the FFT orthonormal).
+        - ``'forward'``: normalize by ``1/n``.
+
+        where ``n`` equals ``prod(s)``, the logical FFT size.
+
+        Default: ``'backward'``.
+
+    Returns
+    -------
+    out: array
+        an array transformed along the axes (dimension) indicated by ``axes``. The returned array must have a complex floating-point data type determined by :ref:`type-promotion`.
+    """
+
+
+def irfftn(x: array, /, *, s: Sequence[int] = None, axes: Sequence[int] = None, norm: Literal['backward', 'ortho', 'forward'] = 'backward') -> array:
+    """
+    Computes the n-dimensional inverse of ``rfftn``.
+
+    Parameters
+    ----------
+    x: array
+        input array. Should have a complex floating-point data type.
+    s: Sequence[int]
+        size of each transformed axis of the **output**. If
+
+        - ``s[i]`` is greater than the size of the input array along a corresponding axis (dimension) ``j``, the corresponding axis of the input array must be zero-padded such that the axis has size ``s[i]``.
+        - ``s[i]`` is less than the length of the input array, each axis ``i`` of the input array must be trimmed to the length ``s[i]``. Except for the last axis is trimmed to ``2 * (m - 1)``, where `m` is the length of the input along the axis.
+
+        If not provided, the length of the transformed axes of the output must equal the length of the input along the axes specified by ``axes``. Default: ``None``.
+    axes: Sequence[int]
+        axes over which to compute the inverse Fourier transform. If not specified, the last ``len(s)`` axes are used, or all axes if ``s`` is not specified either. Default: ``None``.
+    norm: Literal['backward', 'ortho', 'forward']
+        normalization mode. Should be one of the following modes:
+
+        - ``'backward'``: normalize by ``1/n``.
+        - ``'ortho'``: normalize by ``1/sqrt(n)`` (i.e., make the FFT orthonormal).
+        - ``'forward'``: no normalization.
+
+        where ``n`` equals ``prod(s)``, the logical FFT size.
+
+        Default: ``'backward'``.
+
+    Returns
+    -------
+    out: array
+        an array transformed along the axes (dimension) indicated by ``axes``. The returned array must have a real-valued floating-point data type determined by :ref:`type-promotion`. The length along the last transformed axis is ``s[-1]`` (if given) or ``2 * (m - 1)``, and all other axes ``s[i]``.
+    """
+
+
+def hfft(x: array, /, *, n: Optional[int] = None, axis: int = -1, norm: Literal['backward', 'ortho', 'forward'] = 'backward') -> array:
+    """
+    Computes the one-dimensional discrete Fourier transform of a signal with Hermitian symmetry.
+
+    Parameters
+    ----------
+    x: array
+        input array. Should have a real-valued floating-point data type.
```

Comment:
This is wrong (so is the table in https://github.com/data-apis/array-api/pull/189#issuecomment-1201664413). `hfft` is basically `irfft` (C2R transform), so it can be complex.